src/index.js

```diff
@@ -0,0 +1,79 @@
+import 'lodash';
+import './style.css';
+import elementChanges from './check.js';
+import icon from './delete.png';
+import removeAll from './remove.js';
+import Add from './add.js';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
+let title = JSON.parse(localStorage.getItem('title'));
+if (!ToDoArray.length) {
+  document.querySelector('.Activity-list').classList.add('hidden');
+} else {
+  document.querySelector('.Activity-list').classList.remove('hidden');
+}
+document.querySelector('#whatToDo').value = title;
+
+function CreateOb({ description, index }) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+function displayOnScreen(element) {
+  const ul = document.querySelector('.form_');
+  const li = document.createElement('li');
+  li.classList.add('toDo');
+  const checkbox = document.createElement('input');
+  const br = document.createElement('br');
+  const image = document.createElement('img');
+  image.setAttribute('src', icon);
+  image.style.width = '1rem';
+  image.style.float = 'right';
+  image.style.marginTop = '7px';
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.setAttribute('name', 'check');
+  li.style.padding = '9px';
+  const input = document.createElement('input');
+  input.setAttribute('type', 'text');
+  const label = document.createElement('label');
+  label.setAttribute('for', 'check');
+  input.addEventListener('change', () => {
+    element.description = input.value;
+    localStorage.setItem('ToDoArray', JSON.stringify(ToDoArray));
+  });
+  input.value = element.description;
```

Comment:
Look at the image below
- [ ] Your edit functionality does not work very well. When I try to make an edit, the localStorage is cleared. Kindly fix this.

![meq-edit](https://user-images.githubusercontent.com/36542986/144607453-39c9783d-a805-4e1e-bfcb-fbc1878df7c3.jpg)